components/docs-chef-io/content/automate/reports.md

```diff
@@ -13,28 +13,26 @@ draft = false
 
 [\[edit on GitHub\]](https://github.com/chef/automate/blob/master/components/docs-chef-io/content/automate/reports.md)
 
-The Reports view under the Compliance tab provides comprehensive insight into the compliance status of all scanned infrastructure.
+The _Reports_ page (Compliance > Reports) provides comprehensive insight into the compliance status of all scanned infrastructure.
+Scan results for audit cookbook configurations also appear in this view.
 
-{{< note >}}
-Note: The count of controls listed in the _Profile Status_ pane of the Reports Overview represents how many controls have been executed, and not the count of unique controls executed.
-{{< /note >}}
+{{% info %}}
+The control count in the _Profile Status_ pane (Reports > Profile Status) shows the total number of executed controls, not the total number of unique controls.
+{{% /info %}}
 
-![Compliance](/images/automate/compliance.png)
+![Reports](/images/docs/reports.png)
 
 ### Dates in Compliance Reports
 
 The dashboard shows the results of all scans with end times on the _currently selected day_.
 The currently selected day, as highlighted in the search bar, bases itself on timestamps in Coordinated Universal Time (UTC).
```

Comment:
```suggestion
The _currently selected day_, as highlighted in the search bar, bases itself on timestamps in Coordinated Universal Time (UTC).
```

@vjeffrey or @susanev Is this a Unix epoch timestamp or something else?

Reply:
@mjingle use the existing language & make a card to follow up on the timestamps next week.

Reply:
card made

Reply:
Received confirmation from VJ that _current selected day_ is a RFC3339 formatted timestamp.